common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +314,28 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    if not settings.FEATURES.get('MILESTONES_APP', False):
+        return None
+    from milestones.exceptions import (
+        InvalidMilestoneRelationshipTypeException,
+        InvalidCourseKeyException,
+        InvalidUserException,
+    )
+    try:
+        return bool(
+            get_course_milestones_fulfillment_paths(course_id, {"id": user_id})
+        )
+    # Exception when milestone is not found
+    except InvalidMilestoneRelationshipTypeException:
```

Comment:
I think you can achieve the same outcome with slightly more concise code using a list here?  Something like: 

except (Ex1, Ex2, Ex3):
   return False